src/avfoundation.cs

```diff
@@ -2401,6 +2409,14 @@ interface AVAudioSessionDataSourceDescription {
 		
 	}
 
+	[Watch (7,4), NoTV, NoMac, iOS (14,5)]
+	[Native]
+	public enum AVAudioSessionInterruptionReason : ulong {
+		Default = 0,
+		AppWasSuspended = 1,
+		BuiltInMicMuted = 2,
+	}
```

Comment:
pretty sure I've seen this in @rachelkang PR
AVFAudio is part of AVFoundation

Reply:
Oops 

![368C609F-49FA-4B86-94E3-D5AB97CA2449](https://user-images.githubusercontent.com/204671/111723556-18c55b80-883a-11eb-9264-f5d7f1ad386a.gif)